kod/object/passive/spell/reveal.kod

```diff
@@ -119,10 +119,9 @@ messages:
       %% on something with no hope of revealing anything.
       piAdvance = send(oTarget,@RevealHiddenAttributes);
       
-      % Show newly revealed color
+      % Show newly revealed attributes
       if piAdvance
       {      
-         Send(oTarget,@RevealHiddenColor);
```

Comment:
Note: This is removed because my PR calls `RevealHiddenColor()` from `RevealHiddenAttributes()`.